app/controllers/alchemy/pages_controller.rb

```diff
@@ -104,7 +104,7 @@ def load_index_page
     # @return NilClass
     #
     def load_page
-      @page ||= Page.contentpages.find_by(
+      @page ||= Language.current.pages.find_by(
         urlname: params[:urlname],
         language_code: params[:locale] || Language.current.code
```

Comment:
And I think we can remove this, if the pages already are scoped to current language there is no need in also check for the code being the same.